src/extension.ts

```diff
@@ -11,107 +12,55 @@ interface Params {
 
-        if (sourcegraph.app.activeWindowChanges && sentryProjects && sentryProject) {
+        if (sourcegraph.app.activeWindowChanges) {
             const activeEditor = from(sourcegraph.app.activeWindowChanges).pipe(
                 filter((window): window is sourcegraph.Window => window !== undefined),
                 switchMap(window => window.activeViewComponentChanges),
                 filter((editor): editor is sourcegraph.CodeEditor => editor !== undefined)
             )
-
             // When the active editor changes, publish new decorations.
             context.subscriptions.add(
                 activeEditor.subscribe(editor => {
-                    decorateEditor(editor, sentryProject.projectId, sentryProject.lineMatches)
+                    sentryProject
+                        ? decorateEditor(editor, sentryProject.projectId, sentryProject.lineMatches)
+                        : decorateEditor(editor)
                 })
             )
         }
     })
 }
 
-/**
- * Extract Sentry params from document URI necessary to
- * check if the current document sends log events to Sentry and
- * use these params build URL to the Sentry issues stream page.
- * @param textDocument A document URI.
- * @returns repo and file part of URI.
- */
-export function getParamsFromUriPath(textDocument: string): Params {
-    // TODO: Support more than just GitHub.
-    const repoPattern = /github\.com\/([^\?\#]+)/gi
-    const filePattern = /#([^\?\#\/]+)\/.*\.?$/gi
-    const repoM = repoPattern.exec(textDocument)
-    const fileM = filePattern.exec(textDocument)
-    return {
-        repo: repoM![1],
-        file: fileM![0],
-    }
-}
-
-/**
- * Verify if the params from the document URI match with the repo and file formats specified
- * in the Sentry extension settings. If there is a match we know the document is enabled to send logs
- * to Sentry and can send back the corresponding Sentry project ID.
- * @param params params extracted from the document's URI.
- * @param projects Sentry extension projects configurations.
- * @return Sentry projectID this document reports to.
- */
-export function matchSentryProject(params: Params, projects: Settings['sentry.projects']): SentryProject | undefined {
-    // Check if a Sentry project is associated with this document's repo and retrieve the project.
-    const project = projects!.find(p => !!new RegExp(p.patternProperties.repoMatch).exec(params.repo!))
-    if (!project) {
-        return
-    }
-
-    // Check if document matches the file matching pattern specified under the Sentry extension configuration.
-    const fileMatched: boolean = !!project.patternProperties.fileMatches.find(
-        pattern => !!new RegExp(pattern).exec(params.file!)
-    )
-    if (!fileMatched) {
+function decorateEditor(editor: sourcegraph.CodeEditor, sentryProjectId?: string, lineMatches?: RegExp[]): void {
+    if (!editor) {
```

Comment:
`activeEditor` has this:

```typescript
filter((editor): editor is sourcegraph.CodeEditor => editor !== undefined)
```
And the `editor` argument here is typed as `editor: sourcegraph.CodeEditor`, not `editor: sourcegraph.CodeEditor | null`. You shouldn't need this check, have you run into cases where you did need it?